catalog/spatial/geocoding/spatial-geocoding-localsearch/src/test/java/org/codice/ddf/spatial/geocoder/geonames/TestGeoNamesLocalIndex.java

```diff
@@ -118,4 +120,77 @@ public void testGetNearbyCities() throws ParseException, GeoEntryQueryException
 
         assertThat(returnedNearbyLocation, equalTo(mockNearbyLocation));
     }
+
+    @Test
+    public void testGetNearbyCitiesNoResult() throws ParseException, GeoEntryQueryException {
+        NearbyLocation mockNearbyLocation = mock(NearbyLocation.class);
+        when(mockNearbyLocation.getCardinalDirection()).thenReturn("W");
+        when(mockNearbyLocation.getDistance()).thenReturn(10.24);
+        when(mockNearbyLocation.getName()).thenReturn("The City");
+
+        List<NearbyLocation> nearbyLocations = mock(List.class);
+        when(nearbyLocations.size()).thenReturn(0);
+
+        when(geoEntryQueryable.getNearestCities("POINT(1.0 20)",
+                50,
+                1)).thenReturn(nearbyLocations);
+        NearbyLocation returnedNearbyLocation = geoNamesLocalIndex.getNearbyCity("POINT(1.0 20)");
+
+        assertNull(returnedNearbyLocation);
+    }
+
+    @Test
+    public void testGetNearbyCitiesParseException() throws ParseException, GeoEntryQueryException {
+        when(geoEntryQueryable.getNearestCities("POINT(1.0 20)",
+                50,
+                1)).thenThrow(new ParseException("", 1));
+        NearbyLocation returnedNearbyLocation = geoNamesLocalIndex.getNearbyCity("POINT(1.0 20)");
+        assertNull(returnedNearbyLocation);
+    }
+
+    @Test
+    public void testGetCountryCode() throws ParseException, GeoEntryQueryException {
+        when(geoEntryQueryable.getCountryCode("POINT (1.0 2.0)", 50)).thenReturn(Optional.of("US"));
+
+        Optional<String> countryCode = geoNamesLocalIndex.getCountryCode("POINT (1.0 2.0)", 50);
```

Comment:
Can this optional be made a global?  At the very least `POINT (1.0 2.0)` can be made into a constant